src/main/java/core/mate/academy/producer/TruckProducer.java

```diff
@@ -0,0 +1,20 @@
+package core.mate.academy.producer;
+
+import core.mate.academy.model.Truck;
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
+    @Override
+    public List<Truck> get() {
+        Truck truck1 = new Truck();
+        Truck truck2 = new Truck();
+        Truck truck3 = new Truck();
```

Comment:
numbers in variable names - not fixed from the previous review

Reply:
Sorry. I was blind. =)